sql-injection-demo/README.md

```diff
@@ -0,0 +1,64 @@
+# SQL Injection Demo
+This is a demo of how Pixie can be used to capture SQL injections on an application. In
+This demo, we will spin up a
+[DVWA web application](https://hub.docker.com/r/vulnerables/web-dvwa) that is vulnerable
+to SQL injection monitored by Pixie, run
+[SQLMap](https://github.com/SQLMapproject/SQLMap) (a sql injection tool) against that
+application, and detect the SQL injections at the database level  using a PxL script.
+
```

Comment:
I think we need a strongly worded disclaimer at the top resembling DVWA's that this should NOT be applied to a live web server or real environment. 

We should also probably have the instructions include starting up minikube, copying and pasting each step from https://docs.px.dev/installing-pixie/setting-up-k8s/minikube-setup/ (rather than linking to it) so they don't miss it 

wdyt?

Reply:
I've added the instructions for minikube. I wasn't sure if the kubectrl instructions would change at all with minikube or not? I've also added you as a contributor to the forked repo if you want to make any changes to them since I'm not too sure on this part.